### PR TITLE
feat: Export keystore

### PR DESCRIPTION
**Pull request by cvphat**
For #563

### CODE REVIEW

File: lib/services/manager_api.dart [hunk 1]
```diff
@@ -129,9 +129,14 @@ class ManagerAPI {
     }
   }
 
+  Future<File> getKeyStoreFile() async {
```
Comment (oSumAtrIX) [inline lib/services/manager_api.dart:132]: When this is merged, the previous Keystore file will be lost. Migration code is required.

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: lib/services/manager_api.dart (hunk 1 above)
Comment (cvphat) [reply lib/services/manager_api.dart:132]: I have no idea to migrate the previous keystore. This code is just created from `initialize` of `patcher_api.dart` to make sure patcher, import, export and delete is using from the same file

Comment (oSumAtrIX) [reply lib/services/manager_api.dart:132]: If we merge this code, will the previously saved keystore remain in tact?

Comment (cvphat) [reply lib/services/manager_api.dart:132]: Yes

Comment (oSumAtrIX) [reply lib/services/manager_api.dart:132]: How does it guarantee that? The previous location is `/sdcard/Android/data/app.revanced.manager.flutter/files/revanced-manager.keystore` but the new method `getKeyStoreFile()` uses `getTemporaryDirectory()` or `getExternalStorageDirectory()`. Does `getExternalStorageDirectory()` point to `/sdcard/Android/data/app.revanced.manager.flutter/files/`?

Comment (cvphat) [reply lib/services/manager_api.dart:132]: 1. `/sdcard/Android/data/app.revanced.manager.flutter/files/` only used in `deleteKeystore` method
2. the result of `getExternalStorageDirectory()` is `/storage/emulated/0/Android/data/app.revanced.manager.flutter/files` in my phone. it is same with `/sdcard/Android/data/app.revanced.manager.flutter/files`

Comment (Aunali321) [reply lib/services/manager_api.dart:132]: https://github.com/revanced/revanced-manager/tree/custom-keystore

**Comment by Aunali321** [comment]
How is this different from https://github.com/revanced/revanced-manager/tree/custom-keystore?

**Comment by oSumAtrIX** [comment]
Superseded by #776.

**Pull request closed without merging**